README.md

```diff
@@ -15,12 +21,105 @@ This repo uses submodules to pair the [Dark Forest Ethereum backend](https://git
 - Open a terminal tab and run `yarn start`, which will 1) start a local node, 2) deploy the contracts, and 3) run the local client in dev mode
 - When finsihed, the process should pop up your browser to the game client at http://localhost:8081/
 
-You won't have a webserver to drip you a few cents to start playing (which Dark Forest usually does) so instead of creating a new burner wallet, you'll want to import one of the private keys the node funded for you. See the node page where it prints wallets like:
+You won't have a webserver to drip you a few cents to start playing (which Dark Forest usually does)
+so instead of creating a new burner wallet, you'll want to import one of the private keys the node
+funded for you. See the node page where it prints wallets like:
```

Comment:
```suggestion
You won't have a webserver to drip you a few cents to start playing (which Dark Forest usually does) so instead of creating a new burner wallet, you'll want to import one of the private keys the node funded for you. See the node page where it prints wallets like:
```